src/components/accession2/Accession2View.tsx

```diff
@@ -73,6 +74,13 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
     textDecoration: 'none',
   };
 
+  const checkInAccession = async () => {
+    if (accession) {
+      await checkIn(accession.id);
```

Comment:
we should probably show a spinner somewhere while this is happening? Or at least some modal saying checking-in done - click ok. Otherwise the user will barely notice what happened.
I will ask in figma.

Reply:
we can follow up separately maybe. there are no check-in design screens.

Reply:
I guess it's just like this because in the previous flow it was also like this (no modal or spinner)

Reply:
> I guess it's just like this because in the previous flow it was also like this (no modal or spinner)

Yes, I understand. I asked a question on Figma.. we can follow up later if they respond.